libs/components/lists/src/lib/modules/filter/filter-summary-item.component.scss

```diff
@@ -1,16 +1,21 @@
-@use 'libs/components/theme/src/lib/styles/mixins' as mixins;
-@use 'libs/components/theme/src/lib/styles/variables' as *;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-host-overrides() {
+  --sky-override-filter-summary-item-margin-right: 5px;
+  --sky-override-filter-summary-item-padding: 0;
+}
 
 .sky-filter-summary-item {
   display: inline-block;
+  padding: var(
```

Comment:
We can punt this if needed - but it feels a little odd to me that we add this bottom padding no matter if there is a second row or not. Seems like this really should be a `row-gap` property on the `.sky-filter-summary-items` class with this token. That way the bottom padding is only added when needed.